### ISSUE TITLE
Sensor

### ISSUE THREAD

**Issue by kikiloaw** [MartinStokroos/openMicroInverter#2]
Hello again
I found the sensor in your schematic but very expensive, i cant afford.  How can i implement the acs712 sensor 30 amps.  Ang also using voltage divider as voltge sensor, i just know how to revise the schematic but in the code i am not that expert. You are using truerms library that i dont know how to implement. 
Thanks in advance.

**Comment by MartinStokroos** [comment]
I hope my answer given to you under TrueRMS lib satisfies.

**Comment by kikiloaw** [comment]
last question about the anti islanding? is this capable for anti islanding?
also

const float vBattRange = 1.00; 
const float iBattRange = 10.00;
does this settings is thesame as the approach below ? its a bit confusing since the vbatrange only 1.0v
const float outputVoltRange = 660.0; //Vp-p full scale.
const float outputCurrRange = 4.5; //Ap-p full scale.

schematic is incomplete. cnat find where the xor_ouput connected, pll_lock
icant complete the schematic to layout.

**Comment by MartinStokroos** [comment]
The islanding or breaker relays are already put in the development model schematic. I used it with some experiments. The islanding function is far from ready in the software. The current  example sketches do not allow grid connected operation (inverter parallel to grid) !
Yes, vBattRange and iBattRange use the same approach, but these are for scaling the DC measurement ranges.
Current schematic is meant to support software development. Final circuit is under construction and has to be released. 
The XOR_output will become obsolete. I used it for connecting an external PLL-loop filter. In future sketches, the frequency locking will be done completely in software. PLL_lock is just the planned locking indicator, the Arduino LED on pin 13.
Using the current plan will need extra design effort from the user.
